### DISCUSSION TITLE
vscode 打开 xmake create创建的项目 无法生成compile_commands.json

### DISCUSSION THREAD

**Discussion by Smalldy** [xmake-io/xmake#2419] [Q&A] [answered]
安装vscode xmake插件后 打开了之前使用命令行创建的项目 发现没有compile_commands文件。而直接在vscode使用插件 Create Project功能则可以正确生成此文件。 这两者有什么不同吗？

**Comment by Smalldy** [comment, marked as answer]
或者说如何更新此文件呢

**Reply by waruqi** [reply]
xmake.lua 有改动会自动更新生成
